modules/nodes-pods-vertical-autoscaler-about.adoc

```diff
@@ -14,7 +14,7 @@ Recommender::
 The VPA recommender monitors the current and past resource consumption. Based on this data, the VPA recommender determines the optimal CPU and memory resources for the pods in the associated workload object.
 
 Updater::
-The VPA updater checks if the pods in the associated workload object have the correct resources. If the resources are correct, the updater takes no action. If the resources are not correct, the updater kills the pod so that pods' controllers can re-create them with the updated requests.
+The VPA updater checks if the pods in the associated workload object have the correct resources. If the resources are correct, the updater takes no action. If the resources are not correct, the updater ensures that the pods have the updated requests, depending on the VPA update mode, as described in _About the Vertical Pod Autoscaler Operator modes_ later in this section.
```

Comment:
`depending on the VPA update mode`. Whatever update mode that applues updates the user uses, it will ensure pods have the updated requests. I think it might just be better to have a new sentence saying something like

```
The way in which the VPA actuates updates depends on the update mode as described in ...etc.
```

but better worded.